firmware/console/status_loop.cpp

```diff
@@ -689,9 +689,6 @@ void updateTunerStudioState(TunerStudioOutputChannels *tsOutputChannels DECLARE_
 
 	float engineLoad = getEngineLoadT(PASS_ENGINE_PARAMETER_SIGNATURE);
 
-	// header
-	tsOutputChannels->tsConfigVersion = TS_FILE_VERSION;
```

Comment:
cannot do this yet due to console. need to document console dependency

Reply:
reduced to just ini changes for now